torchxrayvision/datasets.py

```diff
@@ -1055,7 +1063,9 @@ def __init__(self, imgpath,
                  xmlpath=os.path.join(datapath, "NLMCXR_reports.tgz"), 
                  dicomcsv_path=os.path.join(datapath, "nlmcxr_dicom_metadata.csv.gz"),
                  tsnepacsv_path=os.path.join(datapath, "nlmcxr_tsne_pa.csv.gz"),
-                 filter_pa=True,
+                 use_tsne_view=False,
```

Comment:
I think this name is more clear:
```suggestion
                 use_tsne_derived_view=False,
```